gerrit/projects/project.py

```diff
@@ -35,6 +35,9 @@ def __init__(self, gerrit_con, name):
         self.branches = project_info.get('branches')
         self.web_links = project_info.get('web_links')
 
+    def __eq__(self, other):
+        return self.name == other.name
```

Comment:
Python doesn't know how to compare two projects porperly if I don't do this.
My assertion that created_project is the same as gotten_project failes like this:
`nose.proxy.AssertionError: <gerrit.projects.project.Project object at 0x7f402fa7e2b0> != <gerrit.projects.project.Project object at 0x7f402fa7e588>`
It should be safe to compare the names of the projects as they are unique.